src/DynamoCore/Graph/Workspaces/SerializationConverters.cs

```diff
@@ -177,15 +177,15 @@ public override bool CanWrite
     /// a NodeFactory, and a Scheduler. These must be supplied at the time of 
     /// construction and should not be serialized.
     /// </summary>
-    public class WorkspaceConverter : JsonConverter
+    public class WorkspaceReadConverter : JsonConverter
```

Comment:
@mjkkirschner It's not being split in two. The workspace serialization doesn't need to use this converter. Trying to do so required references to dynamo model objects like the scheduler, which is not allowed in workspace. I removed this converter from the logic in `ToJson(...)`, and changed the name to make it explicit that it's a converter for reading only.